experiment/runner.py

```diff
@@ -65,9 +65,15 @@
 
 
 def _clean_seed_corpus(seed_corpus_dir):
-    """Moves seed corpus files from sub-directories into the corpus directory
-    root. Also, deletes any files that exceed the 1 MB limit."""
-    if not os.path.exists(seed_corpus_dir):
+    """Prepares |seed_corpus_dir| for the trial. This ensures that it can be
+    used by AFL which is picky about the seed corpus. Moves seed corpus files
+    from sub-directories into the corpus directory root. Also, deletes any files
+    that exceed the 1 MB limit. If the NO_SEEDS env var is specified than the
+    seed corpus files are deleted."""
+    if environment.get('NO_SEEDS'):
```

Comment:
Where are NO_SEEDS and NO_DICTIONARIES set ?

Reply:
They are set in scheduler.py which creates a startup script that runs the container.
This is the same as most other env vars this file uses.

Reply:
You need to add it in startup script template file like
-e NO_SEEDS={{no_seeds}} \
-e NO_DICTIONARIES={{no_dictionaries}} \

Also, no_dictionaries can be shortened to no_dicts everywhere.

Reply:
Ah good catch. Forgot to commit this.
I think the extra letters is fine, I'd rather use dictionaries since no-dictionaries will be user facing.